tests/paren_space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,802 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`arrays.js 1`] = `
+====================================options=====================================
+parenSpacing: false
+parsers: ["babel"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
```

Comment:
How to read this file: This is a set of blocks `input` and `output`. When given the `input` entry, prettier will transform it to the next `output` block.